### PR TITLE
0xb72ED62522783651968ACE0587A0eAdCEEefA074 Contract Logo Upload

### PR DESCRIPTION
**Pull request by VikramCorleone**
Hi! I'm Vikram Thirupathi, developer of the Token Hit Protocol, a decentralized token for deFi. Please upload my token's logo as soon as possible. Me and my team are planned to develop our token by raising funds. Thank You. Have a great day.

### CODE REVIEW

File: blockchains/smartchain/assets/0xb72ED62522783651968ACE0587A0eAdCEEefA074/info.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (Miniiii1989) [inline blockchains/smartchain/assets/0xb72ED62522783651968ACE0587A0eAdCEEefA074/info.json:1]: blockchains/smartchain/assets/0xb72ED62522783651968ACE0587A0eAdCEEefA074/info.json

File: blockchains/smartchain/assets/Hit Protocol/info.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (Miniiii1989) [inline blockchains/smartchain/assets/Hit Protocol/info.json:1]: blockchains/smartchain/assets/Hit Protocol/info.json

**Review by Miniiii1989 — COMMENTED** [review]
0cdd19e8a4c4f6ebbedd12411419206f16b791a4

**Comment by Cryptocool1** [comment]
Please create new PR, follow doc https://developer.trustwallet.com/listing-new-assets/requirements
You may want to try our Assets app in beta; go to https://assets.trustwallet.com, log on with Github, enter contract ID, logo and some additional info, and it will create a PR for you.
See https://developer.trustwallet.com/listing-new-assets/repository_details

**Pull request closed without merging**